common-preferences-bundle.yaml

```diff
@@ -232,6 +344,13 @@ spec:
 apiVersion: instancetype.kubevirt.io/v1alpha2
 kind: VirtualMachinePreference
 metadata:
+  annotations:
+    iconClass: icon-windows
+    openshift.io/display-name: Microsoft Windows 10
```

Comment:
Would it make sense to be able to distinguish the regular windows preferences and the windows virtio preferences by their display name?

Reply:
True, added thanks.